crates/agglayer-config/src/rpc.rs

```diff
@@ -77,11 +88,18 @@ fn default_body_size() -> u32 {
 }
 
 /// The default port for the local RPC server.
-/// If the `PORT` environment variable is set, it will take precedence over
+/// If the `AGGLAYER_PORT` environment variable is set, it will take precedence
+/// over
 fn default_port() -> u16 {
-    from_env_or_default("PORT", DEFAULT_PORT)
+    from_env_or_default("AGGLAYER_PORT", DEFAULT_PORT)
 }
 
+/// The default port for the local admin RPC server.
+/// If the `AGGLAYER_DEBUG_PORT` environment variable is set, it will take
+/// precedence over
+fn default_admin_port() -> u16 {
+    from_env_or_default("AGGLAYER_DEBUG_PORT", DEFAULT_ADMIN_PORT)
+}
```

Comment:
`"AGGLAYER_DEBUG_PORT"` -> `"AGGLAYER_ADMIN_PORT"`. This should be a named constant to avoid the duplicate strings.